optuna_dashboard/ts/action.ts

```diff
@@ -587,17 +589,21 @@ export const actionCreator = () => {
   }
 
   const updatePreference = (
-    study_id: number,
+    studyId: number,
     candidates: number[],
     clicked: number
   ) => {
-    reportPreferenceAPI(study_id, candidates, clicked).catch((err) => {
-      const reason = err.response?.data.reason
-      enqueueSnackbar(`Failed to report preference. Reason: ${reason}`, {
-        variant: "error",
+    reportPreferenceAPI(studyId, candidates, clicked)
+      .then(() => {
+        updateStudyDetail(studyId)
```

Comment:
Could you avoid to send api request in this block? I think you can update preferences without an additional api call.